### PR TITLE
added a gantt table component named jz-gantt

### PR DESCRIPTION
**Pull request by jeremyjone**
- [x] Pull request template structure not broken

### `Type`

> ℹ️  What types of changes does your code introduce?

> 👉 _Put an `x` in the boxes that apply_

- [ ] Fix
- [x] Feature 

### `Checklist`

> ℹ️  Check all checkboxes - this will indicate that you have done everything in accordance with the rules in [CONTRIBUTING](https://github.com/vuejs/awesome-vue/blob/master/.github/contributing.md).

> 👉  _Put an `x` in the boxes that apply._

- [x] Title as described
- [x] Make sure you put things in the right category!
- [x] Always add your items to the end of a list

#### `Open Source`

- [x] Link description does not contain a link to an author / third-party resource
- [x] The documentation (README) contains a description of the project, illustration of the project with a demo or screenshots and a CONTRIBUTING section
- [x] The documentation is in English.
- [x] The project is active and maintained.
- [x] The project accepts contributions.
- [x] Not a commercial product

#### `Apps/Websites`

- [ ] The website is available without errors or ssl certificate problems, and load in a reasonable amount of time.
- [ ] The website is using vuejs intensively. It should detect vue with [vue-devtools](https://github.com/vuejs/vue-devtools).
  > If you cannot detect vue with `vue-devtools` due to work at non public pages (e.g. for enterprise website), you can send Pull Request with screenshot that detected it.
- [ ] The website is original and not too simple. For that reason, blogs and simple landing pages are rejected.
- [ ] A commercial product using Vue, provided that guests could reasonably check out how Vue was used (i.e. A headless CMS which uses Vue for the Admin/editor Area and offers a free tier).

### CODE REVIEW

**Review by Scrum — CHANGES_REQUESTED** [review]
## Checklist

- [ ] git: pull request template structure not broken
- [ ] git: pull request template not selected `type`
- [ ] git: pull request template not checked `Checklist`
- [ ] git: empty commit
- [ ] git: link not to repository
- [ ] git: link description contains a link to the author / third-party resource
- [ ] git: link description no `for Vite` or `Vite plugin`
- [x] git: link description short description short description (one sentence in 24 words, no expanded explanation)
- [x] git: terms in the wrong formats (e.g. Typescript over TypeScript)
- [ ] git: broken PR
- [ ] git: no conflicts resolved
- [ ] git: doublicate
- [ ] time: not updated for more than 6 months
- [ ] category: not in its category
- [ ] lists: not the last on the list
- [ ] docs: no documentation
- [ ] docs: non-english documentation
- [ ] project: commercial product
- [ ] project: dead project
- [ ] project: does not accept donat
- [ ] site: no link to the site
- [ ] site: site with errors
- [ ] site: not actively using vuejs
- [ ] site: not original
> Follow the rules [CONTRIBUTING](https://github.com/vuejs/awesome-vue/blob/master/.github/contributing.md)

**Comment by jeremyjone** [comment]
It has been modified as required

File: README.md [hunk 1]
```diff
@@ -1053,7 +1053,7 @@
...
  - [vuejs-smart-table](https://github.com/tochoromero/vuejs-smart-table) - Straight Forward Table Component using the vanilla HTML Table Structure with Sorting, Filtering, Pagination and Selection out of the box.
  - [@myena/vue-table](https://github.com/myENA/vue-table) - Table Components for Client/Server data handling. Filter, sort, paginate, group, expanding details row. Highly customizible via slots for filter, headers, column, pagination, details row.
- - [jz-gantt](https://github.com/jeremyjone/jz-gantt) - A high-performance vue gantt component, support vue2 & vue3, containing multi-layer expanding, dynamic update data, mergable columns, freely drag the progress bar, multi-layer linkage, multiple date display modes switch, etc.
+ - [jz-gantt](https://github.com/jeremyjone/jz-gantt) - A high-performance vue gantt component, which includes highly customizable table columns, dynamic update data, freely drag the progress bar, switch header, etc.
```
Comment (Scrum) [inline README.md:1056]: `vue` to Vue

**Review by Scrum — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jeremyjone) [reply README.md]: It has been modified as required. `vue` to `Vue`

**Comment by jeremyjone** [comment]
It has been modified as required. `vue` to `Vue`

**Review by Scrum — APPROVED** [review]

**Comment by Scrum** [comment]
@jeremyjone need resolve conflicts

**Comment by jeremyjone** [comment]
@Scrum  Conflict resolved. :)

File: README.md [hunk 2]
```diff
@@ -1197,6 +1197,7 @@
 - [eslint-plugin-vue-a11y](https://github.com/maranran/eslint-plugin-vue-a11y) - Static AST checker for accessibility rules on elements in .vue
 - [vue-focus-lock](https://github.com/theKashey/vue-focus-lock) - It is a trap! A lock for a Focus. A11y util for scoping a focus.
 - [vue-a11y-calendar](https://github.com/IBM/vue-a11y-calendar) - Accessible, internationalized Vue calendar.
+<<<<<<< HEAD
```
Comment (Scrum) [inline README.md:1200]: ?

File: README.md [hunk 3]
```diff
@@ -1314,6 +1315,158 @@
  - [easy-circular-progress](https://github.com/dreambo8563/easy-circular-progress) - Easy circular progress component with counting effect 
 
 ### Tooltip
+=======
```
Comment (Scrum) [inline README.md:1318]: ?

File: README.md [hunk 4]
```diff
@@ -1314,6 +1315,158 @@
...
+- [vue-scroll-progress](https://github.com/spemer/vue-scroll-progress) - Simple Vue.js plugin for page scroll progress bar
+- [vue-read-progress](https://github.com/ajerez/vue-read-progress) - Customizable progress bar at the top of the page that shows the scroll progress
+- [easy-circular-progress](https://github.com/dreambo8563/easy-circular-progress) - Easy circular progress component with counting effect
+- [vue-twitter-counter](https://github.com/LucasLeandro1204/vue-twitter-counter) - Counter component inspired in Twitter with Vue
+- [vue-ins-progress-bar](https://github.com/meloalright/vue-ins-progress-bar) - A Vue component of ins-style progress bar 一款 ins 风格的 vue 进度条组件
+- [vue-next-progressbar](https://github.com/jambonn/vue-next-progressbar) - Slim progress bars(1KB gzipped) for Vue 3.
+- [vue3-progress](https://github.com/marcoschulte/vue3-progress) - Fully customizable vue3 plugin to display a progress bar while waiting for something, e.g. http requests
+
+#### Tooltip
+>>>>>>> a8e9ebad2b9da31b6d8aebee4fa0ad6809c7b719
```
Comment (Scrum) [inline README.md:1469]: ?

**Review by Scrum — CHANGES_REQUESTED** [review]
incorrect resolved

**Comment by jeremyjone** [comment]
Sorry, I didn't notice it. I have revised

**Comment by Scrum** [comment]
> Sorry, I didn't notice it. I have revised

Please, create a new PR

**Pull request closed without merging**